UNRELEASED.md

```diff
@@ -24,6 +24,7 @@ Use [the changelog guidelines](https://git.io/polaris-changelog-guidelines) to f
 - Removed transition on tag button hover state [#1337](https://github.com/Shopify/polaris-react/pull/1337)
 - Added `textAlign` prop to Button ([#1576](https://github.com/Shopify/polaris-react/pull/1576))
 - Made `Button` red when given both the `plain` and `destructive` props ([#1603](https://github.com/Shopify/polaris-react/pull/1603))
+- Added support for destructive actions in `Card` and `Card.Section` ([#1622](https://github.com/Shopify/polaris-react/1622))
```

Comment:
```suggestion
- Added support for complex actions in `Card` and `Card.Section` ([#1622](https://github.com/Shopify/polaris-react/1622))
```

Reply:
Not really sure how to describe the actions lol, but they're more than just destructive now

Reply:
Done!